server/config.ts

```diff
@@ -54,7 +54,6 @@ const defaultCSP = {
     'https://wire.com',
     'https://www.google.com',
     'https://*.giphy.com',
-    'https://*.unsplash.com',
```

Comment:
should we consider checking CSP values in other locations as well? env or BE come to mind.